content/docs/import/import-data-assistant.md

```diff
@@ -1,106 +1,102 @@
+- Extension compatibility
+- Region availability
+- Event triggers and other unsupported features
 
-    You can modify any of these settings to suit the needs of your database, such as the host region, autoscaling range, and so on.
+## Import Your Data
 
-    See [Create a project](/docs/manage/projects#create-a-project) for more details about these options.
+Once checks pass, we'll:
 
-  </div>
-  <div style={{ flex: '0 0 45%', margin: '-15px 0' }}>
-    ![create Neon project](/docs/import/migration_create_project.png)
-  </div>
-</div>
+1. Create a new project (if you started from the Projects page) or use your existing project (if you started from the dashboard)
+2. Create a new branch for your imported data
+3. Copy your data automatically
+4. Verify the import completed successfully
 
-## Move data to Neon
+<Admonition type="note">
+During import, your source database remains untouched — we only read from it to create a copy in Neon.
+</Admonition>
 
-Next, we'll send you to the command line. We generate the `pg_dump` and `pg_store` commands, pre-populated with the correct connection strings and required parameters.
+## Next Steps
 
-<div style={{ display: 'flex', justifyContent: 'center'}}>
-  <img src="/docs/import/migration_move_data.png" alt="move data to Neon using pg_dump and pg_restore" style={{ width: '80%', maxWidth: '600px', height: 'auto' }} />
-</div>
+After a successful import:
 
-The `pg_dump` command is populated with your source database:
+1. Find your newly imported database branch on the **Branches** page of your project.
 
-```bash shouldWrap
-pg_dump -Fc -v -d "postgresql://<username>:<password>@<source_host>:<source_port>/<source_database>" -f database.bak
-```
+   ![Branches page showing imported branch](/docs/import/import_data_assistant_branch.png)
 
-The `pg_restore` command uses the connection string for the database in your newly created project in Neon:
+   _Imported branches are typically named with a timestamp, as shown here._
 
-```bash shouldWrap
-pg_restore -v -d "postgresql://<username>:<password>@<destination_host>:<destination_port>/<destination_database>" database.bak
-```
+2. Run some test queries to ensure everything imported correctly.
+3. Switch your connection string to point to your new Neon database.
 
-For more detailed instructions about using these commands, see [Migrate data using pg_dump and pg_restore](/docs/import/migrate-from-postgres).
+</Steps>
 
-## Next Steps
+## Known Limitations
 
-1. **Verify data integrity** by running some queries and checking that tables and data are present as expected in Neon.
-2. **Switch over your application** by updating your connection string to point to Neon. You can find your connection details on your project Dashboard. See [Connect from any application](/docs/connect/connect-from-any-app) for more information.
+- Currently limited to databases **smaller than 10GB** — but support for larger sizes is in progress.
```

Comment:
_we are actively working on supporting bigger workloads, in the meantime you can contact our support if you are willing to migrate bigger database_